components/ClubPage/SocialLink.js

```diff
@@ -1,14 +1,17 @@
+const stripUrl = url => {
+  return url.replace(/^(?:https?:\/\/)?(?:www\.)?/i, '')
```

Comment:
what about `http`?

Reply:
also you probably want to make sure that this is at the start of the url if there's an edge case where http is somewhere later like in a query param? might be better off not using a regex

Reply:
The question mark makes the `s` optional (matches both `http` and `https`). The `^` ensures that this regex only matches the start of the url.

Reply:
![Screenshot from 2020-01-02 17-42-25](https://user-images.githubusercontent.com/2645695/71697922-7b425100-2d87-11ea-876b-ec857f2134ba.png)

I'm no regex expert but it seems to work for me